docs/example/public-hub-content.mdx

```diff
@@ -44,7 +44,7 @@ export SOURCE_REPO_URL="{url}" # e.g. github.com/capactio/hub-manifests
 # The git ref to checkout. It can point to a commit SHA, a branch name, or a tag.
 export SOURCE_REPO_REF="{ref}" # e.g. main
 
-capact install --capact-overrides="hub-public.populator.enabled=true,hub-public.populator.manifestsLocation.repository=${SOURCE_REPO_URL},hub-public.populator.manifestsLocation.branch=${SOURCE_REPO_REF}"
+capact install --capact-overrides="hub-public.populator.enabled=true,hub-public.populator.manifestsLocations[0].repository=${SOURCE_REPO_URL},hub-public.populator.manifestsLocations[0].branch=${SOURCE_REPO_REF}"
 ```
 
```

Comment:
It would be good to mention in this doc that we can use multiple manifest sources. I would suggest adding such note to every Tab:

```suggestion

> **NOTE**: You can provide multiple manifest sources by setting overrides using different indexes of the `hub-public.populator.manifestsLocations`.

```